boards/arm/stm32/photon/Kconfig

```diff
@@ -46,7 +46,7 @@ config PHOTON_WDG_THREAD_PRIORITY
 
 config PHOTON_WDG_THREAD_STACKSIZE
 	int "Watchdog Thread Stacksize"
-	default 1024
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.